model/stacktrace_frame.go

```diff
@@ -43,3 +43,10 @@ func (s *StacktraceFrame) Transform() common.MapStr {
 
 	return m
 }
+
+func (s *StacktraceFrame) Library() bool {
+	if s == nil || s.LibraryFrame == nil {
+		return false
```

Comment:
I am not sure about the handling of nil pointers here, with this handling it signals an `in_app` library in the stacktrace. It depends if we want to count `nil` as `library` or not. @elastic/apm thoughts?

Reply:
I understand that if `LibraryFrame` is `nil` is because we don't know; and if we don't know is better to consider it as `in_app` frame to not hide (maybe relevant) data to the user.

The `if s == nil` part I assume is good standard Go practice; but I don't think we can have `nil` frames with our current spec.

Reply:
sounds reasonable

Reply:
exactly my thinking @jalvz